test/mailers/access_request_mailer_test.rb

```diff
@@ -16,11 +16,7 @@
     let(:role) { Role::DEPLOYER }
     subject { ActionMailer::Base.deliveries.last }
 
-    before do
-      enable_access_request(address_list, prefix)
-    end
-
-    after { restore_access_request_settings }
+    around { |t| enable_access_request address_list, email_prefix: prefix, &t }
```

Comment:
It seems that now you cannot override `address_list` inside tests. That's the reason the build for this PR is failing `handles no configured email address`